Add Python 3 support and fix failing test for Google country filter

flibbertigibbet opened the pull request:
Test disambiguation of a place name with country name; do not assume default country selection.
Fixes a test that fails when the Google geocoder key is set.
Fixes #45.

Add Python 3 support by running `2to3` and adding backwards support for Python 2 in imports. Adds python 3 to test matrix run on Travis.

Closes #16
Connects #43

flibbertigibbet commented:
Ran `2to3` and included the import fix-ups from #43 to run the full test suite on this branch with both python 2 and 3.

hectcastro commented on .travis.yml:
```diff
@@ -4,6 +4,7 @@ sudo: false
 
 python:
   - '2.7'
+  - '3.6'
```
Because this creates a matrix build, we need to add an additional condition to the `on:` configuration under `deploy:` for Python 3.6 so that only that build publishes to PyPi:

```yml
  on:
    tags: true
    python: "3.6"
    distributions: sdist bdist_wheel
    repo: azavea/python-omgeo
```

flibbertigibbet commented:
Added python version to deploy build, and also added empty `requirements.txt` to hopefully fix the [broken Read the Docs builds](https://readthedocs.org/projects/python-omgeo/builds/5837885/) from #39.
